### DISCUSSION TITLE
Every third Wednesday

### DISCUSSION THREAD

**Discussion by rganz** [quartznet/quartznet#1028] [Q&A] [answered]
Is there a possibility to trigger every third Wednesday*? According to my research, standard Cron doesn't support it, but some implementations can handle it by combining day-of-month and day-of-week (see last two comments here https://serverfault.com/a/404412/464162)

*of course "third" could also be "second" or "fourth" and "Wednesday" could be any other day of week

**Comment by philsayersCDS** [comment, marked as answer]
7:10am on every 3rd wednesday:
0 10 7 ? 1/1 WED#3 *

from: www.cronmaker.com

**Reply by rganz** [reply]
Oh, cool! Didn't knew about that syntax. Thanks a lot

**Reply by rganz** [reply]
I just realized, this is not exactly what I need. It seems your expression is not "**on every** third Wednesday" but "**on the** third Wednesday of the month".
I should have it without the month boundary. Let's say it's every _second_ (not _third_ like in the initial example) Wednesday, then it is triggered only once in the month (on the second Wednesday) instead of twice (on the second and fourth Wednesday).

**Reply by lahma** [reply]
Sounds like find a Wednesday and schedule a trigger having that start time and which runs every three week? There are interval triggers too on top of simple trigger.

**Reply by rganz** [reply]
Thanks for your answer. Too bad that Cron syntax doesn't support this case, because I wanted to use a Cron expression string as the only interface between UI and back end. I'll do it by a custom syntax extension now and handle it manually in the back end.

**Reply by lahma** [reply]
Cron can indeed become the golden hammer quite easily and some things are a bit hard to describe with it, so easier to prepare to express more complex things with some aid, like you have in mind.
